shuup/core/utils/price_display.py

```diff
@@ -74,7 +74,7 @@ def _register(self):
 class PriceDisplayFilter(_ContextFilter):
     def __call__(self, context, item, quantity=1, include_taxes=None, allow_cache=True):
         key, val = context_cache.get_cached_value(
-            identifier=self.cache_identifier, item=item, context=context,
+            identifier=self.cache_identifier, item=item, context=context.get('request', context),
```

Comment:
I think it always should have. This is the context from jinja. There is a current code (from master) which gets the request from the context here: https://github.com/shuup/shuup/blob/master/shuup/core/utils/price_display.py#L90
If mine is wrong, then this ^^ is also wrong.